src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -874,6 +909,24 @@ public void AddRequestContext()
                 _parameters.Add(KnownParameters.RequestContext);
             }
 
+            private void AddRequestParameterWithMediaTypes(RequestParameter requestParameter, ICollection<string> requestMediaTypes)
+            {
+                CSharpType type = new CSharpType(typeof(ContentType), requestParameter.IsNullable || !requestParameter.IsRequired);
+                var parameter = new Parameter(
+                    requestParameter.CSharpName(),
+                    CreateDescriptionWithMediaTypes(requestParameter, requestMediaTypes),
+                    TypeFactory.GetInputType(type),
+                    null,
+                    requestParameter.IsRequired,
```

Comment:
@annelo-msft, I think we should always validate this parameter, so we need to pass `true` here.

Reply:
We can always remove validation without a breaking change, so let's go ahead with this and we'll deal with it later if it's a problem.

Reply:
If it's not marked required, it doesn't seem like we should validate it.

Why is this important to us, @AlexanderSher ?

Reply:
@AlexanderSher we will not validate `ContentType`, because it is a structure (value type).